services/browsers/database.go

```diff
@@ -0,0 +1,136 @@
+package browsers
+
+import (
+	"database/sql"
+
+	"github.com/status-im/status-go/services/browsers/migrations"
+	"github.com/status-im/status-go/sqlite"
+)
+
+// Database sql wrapper for operations with browser objects.
+type Database struct {
+	db *sql.DB
+}
+
+// Close closes database.
+func (db Database) Close() error {
+	return db.db.Close()
+}
+
+// InitializeDB creates db file at a given path and applies migrations.
+func InitializeDB(path, password string) (*Database, error) {
+	db, err := sqlite.OpenDB(path, password)
+	if err != nil {
+		return nil, err
+	}
+	err = migrations.Migrate(db)
+	if err != nil {
+		return nil, err
+	}
+	return &Database{db: db}, nil
+}
+
+type Browser struct {
+	ID           string   `json:"id"`
+	Name         string   `json:"name"`
+	Timestamp    uint64   `json:"timestamp"`
+	Dapp         bool     `json:"dapp"`
+	HistoryIndex int      `json:"historyIndex,omitempty"`
```

Comment:
What is `HistoryIndex` and `History` for?

Reply:
history is a vector of URLs and historyIndex tells you which one is the current one. However @dshulyak it would be great if it could be `history-index`, `dapp?`, `browser-id` so we can use the data directly?